references/shacl-20170720.ttl

```diff
@@ -489,7 +489,7 @@ sh:ClassConstraintComponent
 sh:ClassConstraintComponent-class
 	a sh:Parameter ;
 	sh:path sh:class ;
-	sh:nodeKind sh:IRI ;
+	sh:datatype xsd:anyURI ;
```

Comment:
```suggestion
	sh:nodeKind sh:IRI ;
```

I am not sure if this change is correct. sh:IRI is actually semantically correct IMHO. CodeGen should support that, or not?